app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -189,36 +175,38 @@ def actor
 
     ##
     # @return [#errors]
-    def create_work
-      case curation_concern
-      when ActiveFedora::Base
```

Comment:
Moved AF processing to calling method and changes this method to `#create_valkyrie_work` which only handle processing for vaklyrie works.